__tests__/unit/core/parser/component-with-proper-values.json

```diff
@@ -0,0 +1,98 @@
+{
+  "$schema": "https://raw.githubusercontent.com/teleporthq/teleport-code-generators/master/src/uidl-definitions/schemas/component.json",
+  "name": "ComponentWithPrimitiveProps",
+  "propDefinitions": {
+    "test": {
+      "type": "string",
+      "defaultValue": "123"
+    },
+    "content": {
+      "type": "object",
+      "defaultValue": {
+        "heading": "Hello World"
+      }
+    }
+  },
+  "node": {
+    "type": "element",
+    "content": {
+      "elementType": "container",
+
+      "style": {
+        "flexDirection": {
+          "type": "dynamic",
+          "content": {
+            "referenceType":"prop",
+            "id": "test"
+          }
+        },
+        "alignSelft": {"type":"static", "content": "center"}
```

Comment:
alignSelf :)

Reply:
ah maaan, I had like 20 of these. Some slipped the mass rename.

Reply:
:))) another minus for JSON representation